example_sns_init.yaml

```diff
@@ -25,104 +31,307 @@ description: >
+    # You need to replace this for your own SNS decentralization swap.
+    # For testing, propose_sns.sh will fill this in automatically.
+    - YOUR_PRINCIPAL_ID
 
+# The list of dapp canister(s) in the form of canister IDs, for example,
+# `bnz7o-iuaaa-aaaaa-qaaaa-cai`. These dapp canister(s) will be decentralized
+# if the decentralization swap succeeds, and must be co-controlled by NNS Root 
+# (`r7inp-6aaaa-aaaaa-aaabq-cai`) to be properly transferred to the newly
+# created SNS.
 dapp_canisters:
-    # You need to replace these values.
-    - YOUR_CANISTER_ID  # propose_sns.sh can fill this in automatically.
-    # More canisters can be listed here.
+    # You need to replace this for your own SNS decentralization swap.
+    # For testing, propose_sns.sh will fill this in automatically.
+    - YOUR_CANISTER_ID
 
+# Configuration of SNS tokens in the ICRC Ledger canister deployed as part
+# of the SNS.
 Token:
+    # The name of the token issued by the SNS ledger.
+    # Must be a string with a length between 4 and 255 characters.
+    #
+    # Example: InternetComputerProtocol
     name: Rock Out Token
+
+    # The symbol of the token issued by the SNS Ledger.
+    # Must be a string with a length between 3 and 10 characters.
+    #
+    # Example: ICP
     symbol: ROT
-    transaction_fee: 10_000 e8s # You can leave this alone.
+
+    # SNS ledger transactions will have this fee.
+    transaction_fee: 10_000 e8s
+
+    # Path to the SNS token logo on your local filesystem. The path is relative
+    # to the configuration file location, unless an absolute path is given.
+    # Must have less than 341,334 characters, roughly 256 Kb. The only supported
+    # format is PNG.
     logo: logo.png
 
-# It is fine to leave these alone.
+# Configures SNS proposal-related fields. These fields have no
+# effect on the NNS proposal that is created from this file, but will affect all
+# SNS proposals submitted to the newly created SNS.
 Proposals:
+    # The cost of making an SNS proposal that is rejected by the SNS Neuron
+    # holders.
     rejection_fee: 1 token
+
+    # The initial voting period of a newly-created SNS proposal. (A proposal's
+    # voting period may be increased during its lifecycle due to the
+    # wait-for-quiet algorithm (see details below).
     initial_voting_period: 4d
+
+    # The wait for quiet algorithm extends the voting period of a proposal when
+    # there is a flip in the majority vote during the proposal's voting period.
+    #
+    # Without this, there could be an incentive to vote right at the end of a
+    # proposal's voting period, in order to reduce the chance that people will
+    # see and have time to react to that.
+    #
+    # TODO: fix text below
+    # If this value is set to 1 day, then a change in the majority vote at the
+    # end of a proposal's original voting period results in being extended by an
+    # additional day. Another change at the end of the extended
```

Comment:
does "at the end" mean that if the majority vote changes _during_ the extended period and that the majority vote changes back _during_ the extended period (so that there is no change of the majority vote at the end of the extended period w.r.t. the beginning of the extended period), then no further extension is made? maybe it's worth clarifying explicitly

Reply:
I don't know off the top of my head, so I'm adding this to the list of follow-up improvements.